ClemBot.Bot/bot/cogs/trivia_cog.py

```diff
@@ -363,12 +324,12 @@ def chunk_list(lst, n):
 
 DIFFICULTY_LOWER = [k.lower() for k in DIFFICULTY]
 
-QUESTIONTYPE = [
+QUESTION_TYPE = [
     "multiple",
     "boolean"
 ]
 
-CATEGORYLIST = ["General-Knowledge",  #Including this out of consistency to avoid making the offset 10 for no reason. This will be the default value.
+CATEGORY_LIST = ["General-Knowledge",  #Including this out of consistency to avoid making the offset 10 for no reason. This will be the default value.
```

Comment:
These darn spaceless comments amirite
```suggestion
CATEGORY_LIST = ["General-Knowledge",  # Including this out of consistency to avoid making the offset 10 for no reason. This will be the default value.
```